docs/getting-started/tutorial-sample-remove.asciidoc

```diff
@@ -0,0 +1,18 @@
+[[tutorial-sample-remove]]
+=== Wrapping up
+
+When you’re done experimenting with the sample data set, you can remove it. 
+
+. Go to the {kib} home page and click the link next to  *Sample data*. 
+. On the *Sample flight data* card, click *Remove*.
+
+Now that you have a handle on the {kib} basics, you might be interested in:
+
+* <<tutorial-your-data, Getting the Full {kib} Experience>>.  You’ll learn how to load your own 
```

Comment:
So this may be out of scope for this update, but what are your thoughts on either adding or replacing the existing data sets with something a that is actually offered in the product? For instance, we have all of these add data tutorials that we're guiding users to. This list is growing too. Something like systems metrics allow you to follow a few steps in a tutorial to start collecting data from your machine. These come with prepackaged dashboards as well. Is this something we could work into the mix?